src/timer.rs

```diff
@@ -132,8 +131,9 @@ impl Timer<SYST> {
 
 impl CountDown for Timer<SYST> {
     type Time = Hertz;
+    type Error = nb::Error<Error>;
```

Comment:
Does this need to be `nb::Error<Error>`? I think could just be `Error`.

Reply:
the `try_wait` function below returns `nb::Error::WouldBlock` so, without changing this, yes?

Reply:
wait you’re right i think i have this backwards, will try fixing it tomorrow. thanks for the comment!

Reply:
got it, thanks for the review!